dev/run-tests.py

```diff
@@ -305,8 +305,8 @@ def get_hadoop_profiles(hadoop_version):
     """
 
     sbt_maven_hadoop_profiles = {
-        "hadoop2.6": ["-Phadoop-2.6"],
-        "hadoop2.7": ["-Phadoop-2.7"],
+        "hadoop2.6": ["-Phadoop-3.1"],
+        "hadoop2.7": ["-Phadoop-3.1"],
```

Comment:
@jerryshao, Here, replaced the profile to point out `-Phadoop-3.1` by force :-)

Reply:
I see.